src/lib/ThumbnailsSidebar.js

```diff
@@ -59,8 +60,10 @@ class ThumbnailsSidebar {
         this.renderNextThumbnailImage = this.renderNextThumbnailImage.bind(this);
         this.requestThumbnailImage = this.requestThumbnailImage.bind(this);
         this.thumbnailClickHandler = this.thumbnailClickHandler.bind(this);
+        this.onKeydown = this.onKeydown.bind(this);
 
         this.anchorEl.addEventListener('click', this.thumbnailClickHandler);
+        this.anchorEl.addEventListener('keydown', this.onKeydown);
```

Comment:
What's the behavior if we hold down the button?

Reply:
You will experience some lag due to all of the pdf.js renders. We don't currently throttle page changes when using the doc shortcuts, but it's a good optimization going forward.